stratum/hal/bin/np4intel/docker/Dockerfile.builder

```diff
@@ -0,0 +1,132 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BAZEL_VERSION=2.2.0
+ARG JDK_URL=https://mirror.bazel.build/openjdk/azul-zulu11.29.3-ca-jdk11.0.2/zulu11.29.3-ca-jdk11.0.2-linux_x64.tar.gz
+ARG LLVM_REPO_NAME="deb http://apt.llvm.org/stretch/  llvm-toolchain-stretch main"
+ARG PROTOBUF_TAG=v3.7.1
+ARG GRPC_TAG=v1.21.3
+ARG DPDK_VERSION=v19.08-rc4
+ARG JOBS=4
+ARG NP4_TAR
+ARG NP4_VERSION=4.7.1-1
+ARG NP4_DIR=/np4_intel
+
+FROM ubuntu:18.04
+LABEL maintainer="Stratum Ubuntu dev <stratum-dev@lists.stratumproject.org>"
+LABEL description="This Docker image sets up a development environment for Stratum on Ubuntu"
+
+# Copy in the NP4 tarball
+ARG NP4_TAR
+ARG NP4_DIR
+COPY $NP4_TAR /
+RUN mkdir $NP4_DIR && tar xf /$NP4_TAR -C $NP4_DIR --strip-components 1
+
+# bazel dependencies
+# + wget to download bazel binary
+# + ca-certificates for wget HTPPS connection
+# LLVM dependencies
+# + gnupg2 for apt-key
+# + software-properties-common for add-apt-repository
+ENV PKG_DEPS pkg-config zip zlib1g-dev unzip python wget ca-certificates \
+    ssh git gdb vim emacs-nox sudo libudev-dev libjudy-dev bison flex \
+    libfl-dev libgmp-dev libi2c-dev python-yaml libyaml-dev build-essential \
+    lcov curl autoconf automake libtool libgmp-dev libpcap-dev \
+    libboost-thread-dev libboost-filesystem-dev libboost-program-options-dev \
+    gnupg2 software-properties-common python-pip python-dev python3-dev \
+    libfdt1 libnuma-dev libhugetlbfs-dev linux-virtual dkms \
+    libjson-c3 libjson-c-dev cmake libhwloc-dev uuid-dev
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends $PKG_DEPS
+
+# LLVM toolchain
+ARG LLVM_REPO_NAME
+RUN wget --quiet -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN add-apt-repository "$LLVM_REPO_NAME"
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends clang-format clang
+
+ARG BAZEL_VERSION
+RUN wget https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
+    chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
+    ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
+    rm -f bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+
+# Install java and lcov for Bazel coverage
+ARG JDK_URL
+ENV JAVA_HOME /usr/local/lib/jvm
+RUN wget $JDK_URL -O $HOME/jdk_11.0.2.tar.gz && \
+    mkdir -p $JAVA_HOME && \
+    tar xf $HOME/jdk_11.0.2.tar.gz -C $JAVA_HOME --strip-components=1 && \
+    rm $HOME/jdk_11.0.2.tar.gz
+ENV PATH=$PATH:/usr/local/lib/jvm/bin
```

Comment:
Replace with `FROM stratumproject/build:build as builder` ?

Reply:
No, the container is based on ubuntu because Netcope SDK requires either Ubuntu or CentOS for the install scripts to work etc.  Could see if we can get them to support Debian over time given it's pretty close, but for now, that is what we have.

Reply:
Ahh didn't see the FROM line.